### ISSUE TITLE
Make readiness probe checking configurable for Ingress/Routes

### ISSUE THREAD

**Issue by Ferlonas** [stakater/IngressMonitorController#217]
When an ingress defines a path, and the deployment has a readinessProbe path defined (it doesn't matter of the paths are the same or different), the controller will create a monitor for ingress-host/ingress-path/readiness-path. This will result in 404 errors, as the only correct available paths are ingress-host/ingress-path and ingress-host/readiness-path.

Using readinessProbes for monitoring should be configurable in order to avoid this problem.
The overridePath annotation is not a good workaround, as it requires manually adapting the path whenever it changes in the ingress.

On another note: is it correct that the controller will only take the first rule of each ingress?

**Comment by usamaahmadkhan** [comment]
@Ferlonas Yes right now only first rule is used for ingresses. Would you like to contribute a PR for readinessprobe issue?

**Comment by usamaahmadkhan** [comment]
The logic to get the readiness/health path is already there. I think we need to return a list of URLs for  host URL and readiness/health URL. @waseem-h  what do you think? Also this needs to get fixed for openshift routes as well

**Comment by Ferlonas** [comment]
@usamaahmadkhan I might open a PR once I find the time to do so. I'll have to take a look at the code though to see if I can find a good solution.

**Comment by karl-johan-grahn** [comment]
Reopening issues that inadvertently were closed as stale
